### PR TITLE
fix: add instructions for learners

### PR DESCRIPTION
**Pull request by Ian2012**
### Description
This PR adds instructions for students and instructors to use the mind map component.

### How To Test
1. Install this branch in your environment.
2. Access a unit in LMS/Studio.
3. You should see the instructions to use the mind map component as a drop-down menu.
### Screenshots

1. LMS
image
image
4. Studio
image
image

### CODE REVIEW

**Comment by mariajgrimaldi** [comment]
can you rebase with the main branch? thanks!

File: mindmap/static/html/mindmap.html [hunk 1]
```diff
@@ -1,24 +1,66 @@
...
-  <p>{% trans "The mind map cannot be rendered due to the following error" %}: <b>{{error_message}}</b></p>
-  <p>{% trans "Please contact the instructor or an administrator" %}.</p>
-{% else %}
-  <link type="text/css" rel="stylesheet" href="https://unpkg.com/jsmind@latest/style/jsmind.css" />
-  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.11.5/css/jquery.dataTables.css" />
-
-  <div>
-    <div id="jsmind_container_{{xblock_id}}" class="jsmind_container"></div>
+
+{% if editable and can_submit_assignment or not in_student_view %}
```
Comment (mariajgrimaldi) [inline mindmap/static/html/mindmap.html:3]: where did the error message go?

Comment (Ian2012) [reply mindmap/static/html/mindmap.html:3]: I think the error message was added way back when using django storages, now it's not needed anymore. But not sure, as I've not been involved so often in this development

Comment (BryanttV) [reply mindmap/static/html/mindmap.html:3]: Yes, the `error_message` variable is no longer in the context due to the change mentioned by @Ian2012.

**Review by mariajgrimaldi — APPROVED** [review]

**Comment by Ian2012** [comment]
@BryanttV feel free to merge once you bump version and add a changelog entry

**Pull request merged**